azurerm/data_source_kubernetes_cluster.go

```diff
@@ -564,6 +587,26 @@ func flattenKubernetesClusterDataSourceAgentPoolProfiles(input *[]containerservi
 			agentPoolProfile["count"] = int(*profile.Count)
 		}
 
+		if profile.MinCount != nil {
+			agentPoolProfile["min_count"] = int(*profile.MinCount)
+		}
+
+		if profile.MaxCount != nil {
+			agentPoolProfile["max_count"] = int(*profile.MaxCount)
+		}
+
+		if profile.Type != "" {
```

Comment:
This is already set above on line 582

Reply:
Correct. Merge issue, re added in one of the rebases.

Reply:
this was fixed days I go.